### PR TITLE
[INT-1171] Add Copy Pachyderm File URI command

### PR DESCRIPTION
**Pull request by emmajsadams**
Based on this notion doc https://www.notion.so/Copy-Pachyderm-File-URI-from-Jupyterlab-File-Browser-192a4a8a108548759cc197a010ab94cd

It it is pretty similar to the doc, except I forgot I need to add a case that handles lack of clipboard permissions (due to lack of https). There isn't really a great way to work around this so it seems like just asking the users to copy the text from the notification is a good option.

## With Clipboard permissions
Screenshot 2024-04-21 at 8 49 29 PM

## Without Clipboard permissions (only users outside an https context will see this)
Screenshot 2024-04-21 at 8 48 50 PM

### CODE REVIEW

File: jupyter-extension/src/plugins/mount/customFileBrowser.ts [hunk 1]
```diff
@@ -129,10 +130,86 @@ const createCustomFileBrowser = (
...
+        app.commands.addCommand('open-pachyderm-sdk', {
+          execute: () => {
+            window
+              ?.open('https://docs.pachyderm.com/latest/sdk/', '_blank')
+              ?.focus();
+          },
+        });
+      }
+
+      commands.addCommand('open-determined', {
```
Comment (kevyang) [inline jupyter-extension/src/plugins/mount/customFileBrowser.ts:145]: not sure the name here makes the most sense; this feature will have utility outside of determined too

**Review by kevyang — COMMENTED** [review]
can we add some tests for this?

File: jupyter-extension/src/plugins/mount/customFileBrowser.ts (hunk 1 above)
Comment (emmajsadams) [reply jupyter-extension/src/plugins/mount/customFileBrowser.ts:145]: yeah good point, will switch it up!

**Comment by emmajsadams** [comment]
> can we add some tests for this?

I added a test to make sure the context menu pops up. Are you thinking additional unit tests for the functions in the context menu? Can do.

**Comment by kevyang** [comment]
> > can we add some tests for this?
> 
> I added a test to make sure the context menu pops up. Are you thinking additional unit tests for the functions in the context menu? Can do.

Yeah, I was thinking that could be good. But this isn't a huge change and the release is being cut today, we can add the test at a later time.

**Review by kevyang — APPROVED** [review]

**Pull request merged**